### ISSUE TITLE
Creating dataset from random images collected just for inference purpose via a pretrained model.

### ISSUE THREAD

**Issue by pranavchat14** [webdataset/webdataset#74]
How do I create a dataset that does not contain any label information?

I plan to download random images and perform inference on them by using a pre-trained model. 
What should the structure of this dataset be and how should I create a tar file for this to be further utilized in webdataset?

All the examples show use cases of having an image file and an associated target file with the same basename, and that has got me confused as to how to go about creating a tar file just for inference purposes.

**Comment by tmbdev** [comment]
That's really easy: just create a tar file of all the images:

```
tar cf inference-data.tar data/*.jpg
```

For using the data, just use `to_tuple` with only the `jpg` extension (you get a tuple with one element), or just use dictionary-style samples:

```
dataset = wds.WebDataset(...). ... .to_tuple("jpg").batched(64)
loader = wds.WebLoader(dataset, ...)

for images, in loader:
   ...
```

**Comment by pranavchat14** [comment]
Thanks a lot, it worked. For some reason, in most of the places, docstrings are not showing. That would be really helpful.

P.S. Thank you for creating this.
